src/java.base/share/classes/java/lang/Runtime.java

```diff
@@ -119,85 +177,46 @@ public void exit(int status) {
-     * of events:
-     *
-     *   <ul>
-     *
-     *   <li> The program <i>exits</i> normally, when the last non-daemon
-     *   thread exits or when the {@link #exit exit} (equivalently,
-     *   {@link System#exit(int) System.exit}) method is invoked, or
-     *
-     *   <li> The virtual machine is <i>terminated</i> in response to a
-     *   user interrupt, such as typing {@code ^C}, or a system-wide event,
-     *   such as user logoff or system shutdown.
-     *
-     *   </ul>
-     *
-     * <p> A <i>shutdown hook</i> is simply an initialized but unstarted
-     * thread.  When the virtual machine begins its shutdown sequence it will
-     * start all registered shutdown hooks in some unspecified order and let
-     * them run concurrently.  When all the hooks have finished it will then
-     * halt. Note that daemon threads will continue to run during the shutdown
-     * sequence, as will non-daemon threads if shutdown was initiated by
-     * invoking the {@link #exit exit} method.
-     *
-     * <p> Once the shutdown sequence has begun it can be stopped only by
-     * invoking the {@link #halt halt} method, which forcibly
-     * terminates the virtual machine.
-     *
-     * <p> Once the shutdown sequence has begun it is impossible to register a
-     * new shutdown hook or de-register a previously-registered hook.
-     * Attempting either of these operations will cause an
-     * {@link IllegalStateException} to be thrown.
-     *
-     * <p> Shutdown hooks run at a delicate time in the life cycle of a virtual
-     * machine and should therefore be coded defensively.  They should, in
+     * <p> A <i>shutdown hook</i> is simply an initialized but unstarted thread. Shutdown hooks
+     * are started at the beginning of the <a href="#shutdown">shutdown sequence</a>.
+     * Registration and de-registration of shutdown hooks is disallowed once the shutdown
+     * sequence has begun.
+     * <p>
+     * Uncaught exceptions are handled in shutdown hooks just as in any other thread, as
+     * specified in {@link Thread.UncaughtExceptionHandler}. After the uncaught exception
+     * handler has completed, the shutdown hook is considered to have terminated and is not
+     * treated differently from a hook that has terminated without having thrown an
+     * uncaught exception.
+     *
+     * @apiNote
+     * Shutdown hooks run at a delicate time in the life cycle of a virtual
+     * machine and should therefore be coded defensively. They should, in
      * particular, be written to be thread-safe and to avoid deadlocks insofar
-     * as possible.  They should also not rely blindly upon services that may
+     * as possible. They should also not rely blindly upon services that may
      * have registered their own shutdown hooks and therefore may themselves in
-     * the process of shutting down.  Attempts to use other thread-based
+     * the process of shutting down. Attempts to use other thread-based
```

Comment:
Is there a "be" missing here? 
> may themselves *be* in the process ...

Reply:
Yes, "be" is missing. Hm, I hadn't edited that text... Ha that typo was introduced by MR in 1999! OK, fixed.